test/src/element/testCommandsElementSelectors.js

```diff
@@ -155,4 +155,42 @@ describe('test commands element selectors', function() {
 
     Nightwatch.start(done);
   });
+
+  it('waitForElementNotPresent(<{selector}>) success', function(done) {
```

Comment:
Please limit the changes to the nocks. Passing `undefined`/`infinity` as an argument looks a bit irregular so I think it's better to avoid it. I would rather not introduce more parameters to any nocks for now. We might refactor all the nocks at a later point.

Is it necessary to have more than one retry? I don't think it is, but if it is you can simply do:
```
nocks
      .elementsFound('.nock-string')
      .elementsFound('.nock-string')
```

Also try to keep the timeout short - 300ms is way to much for what's needed here, that's why I suggested 50.

Reply:
Basically all nocks call `persist` which means that this response is returned _forever_.
In order to "simulate" a real use I wanted these elements to be found 3 times and after that they are not found anymore.
I'm not sure how I should test this without that. I will have to think about this test again then...

Reply:
Yes, and I'm saying that we don't need to persist the nocks for 3 times, one should be enough - as I did in my initial example. If we persist 3 times we make the test a little longer than needed. Still, if you insist that it should be 3 times, I think you can just repeat the `.elementsFound('.nock-string')` 3 times without introducing any changes in the nocks libraries.

Reply:
You can control the retryInterval by passing an extra numeric argument after the timeout value - this is in order to have a shorter timeout. 

```
.waitForElementNotPresent('#nock-element', 50, 20, ...args)
```
or 
```
.waitForElementNotPresent({selector: '#nock-element', timeout: 50, retryInterval: 20})
```

Reply:
I just realised that all the nocks from `nockselement.js` are persisted. But those from `nocks.js` are not, so I was referring to those.

I would like to release a new version today so if you don't have time I can finish up the tests.

Reply:
I'm afraid I cannot finish it today... feel free to take over.

Reply:
I was able to squeeze it in :)

Reply:
Great, thanks for bearing with me here :)